packages/lesswrong/server/emails/renderEmail.tsx

```diff
@@ -157,9 +163,15 @@ export async function generateEmail({user, to, from, subject, bodyComponent, boi
   // visited since before that feature was implemented.
   const timezone = user?.lastUsedTimezone || null
   
+  const themeOptions: ThemeOptions = {name: "default", siteThemeOverride: {}};
+  const theme = getForumTheme(themeOptions);
+  const stylesContext = createStylesContext(theme);
+  
   const wrappedBodyComponent = (
     <EmailRenderContext.Provider value={{isEmailRender:true}}>
     <ApolloProvider client={apolloClient}>
+    <ThemeContextProvider options={themeOptions}>
+    <FMJssProvider stylesContext={stylesContext}>
     <JssProvider registry={sheetsRegistry} generateClassName={generateClassName}>
```

Comment:
I'm struggling to understand the difference between the stylesContext and the sheetsRegistry. Which may be a me-problem, or a not-this-PRs-problem. But consider putting some documentation somewhere and/or telling me on github.

Reply:
`sheetsRegistry` is a material-UI thing, while `stylesContext` is its replacement. Currently they coexist. When the dust settles with the current round of refactors, the material-UI side should be gone entirely.